app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
||= dùng để làm gì em

Reply:
toán tử ||= e hiểu là @current_user sẽ bằng nil khi mà cả @current_user = nil và User.find_by trả về nil

Reply:
Chỉ nil thôi à em. Vs thế nếu giá trị của @current_user hoặc User.find_by khác nil thì sao nhỉ

Reply:
.

Reply:
e đặt a ||=b cho dễ ạ. với a là biến và b là 1 hàm nào đó. nếu a có giá trị nil hoặc false thì b được thực thi và kết quả được gán vào a.
nếu a khác nil/false thì b sẽ ko thực thi, a giữ nguyên giá trị ạ